package.json

```diff
@@ -21,8 +21,8 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.25.0",
-    "@sveltejs/adapter-auto": "next",
-    "@sveltejs/kit": "next",
+    "@sveltejs/adapter-auto": "1.0.0-next.80",
+    "@sveltejs/kit": "1.0.0-next.505",
```

Comment:
After using `pnpm install` while importing fontawesome, I downloaded the latest versions of these two packages, which caused a massive bug which looked like this each time I tried to start the dev server:

```
failed to load config from /Users/nickspatties/Documents/dev/megaresume_root/megaresume/vite.config.ts error when starting dev server: preprocessCSS
```

It looked like all of a sudden `preprocessCSS` no longer existed. Don't know what that's about.